### PR TITLE
feat: add consent popup support

### PR DESCRIPTION
**Pull request by goshander**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
   },
   "devDependencies": {
     "@diplodoc/babel-preset": "^1.0.2",
-    "@diplodoc/components": "^4.3.0",
+    "@diplodoc/components": "^4.9.2",
```
Comment (martyanovandrey) [inline package.json:46]: better to use latest - 4.9.3, 4.9.2 has some problems

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -188,6 +197,11 @@ export function App(props: DocInnerProps): ReactElement {
                             </PageConstructorProvider>
                         )}
                     </Page>
+                    <ConsentPopup
```
Comment (martyanovandrey) [inline src/components/App/App.tsx:200]: we don't need to show <ConsentPopup when analytics not defined

Comment (goshander) [reply src/components/App/App.tsx]: i add condition for this, but ConsentPopup components already has a check inside

Comment (martyanovandrey) [reply src/components/App/App.tsx]: I checked locally - before you added the condition in this place, the popup was rendered regardless of the presence of the analytics block

**Pull request merged**